source/_integrations/sql.markdown

```diff
@@ -95,6 +95,12 @@ Use the same Database URL as for the `recorder` integration. Change `DB_NAME` to
 
 Example Database URL: `"mssql+pyodbc://username:password@SERVER_IP:1433/DB_NAME?charset=utf8&driver=FreeTDS"`
 
+<div class='note info'>
```

Comment:
How would the average user of Home Assistant do this? Around 96% of our users use a container-based setup (Docker, Supervised, OS), and cannot use the linked instructions.

Reply:
They can't if I understand it correctly.
So I presume you mean I have to clarify this that it only works on non-container setup where you are in control of the operating system to install dependencies?

Reply:
Yup, we have to make clear if they can do it, if so: how. Otherwise, make clear they can't. If we don't provide this clarity, those people will try to do stuff, that won't work and leads to frustrations.